docs/functions/util.rst

```diff
@@ -17,6 +17,20 @@ These functions are designed to be used by themes.
    Stores the *code* in a variable named *variable*. If *code* is not set, uses
    the last return code instead.
 
+.. function:: __lp_bytes_to_human(bytes, [precision]) -> var:lp_bytes, var:lp_bytes_units
+
+   Convert the given bytes into a human-readable, binary-prefixed notation,
+   (i.e. powers of 2, as opposed to decimal-prefixed notation).
```

Comment:
This wording is confusing. I think what you mean is powers of 2 by 10s, so 1024. I would word it more like:
```suggestion
   Convert the given bytes into a human-readable notation, using binary
   multiple-byte units. Eg: KiB instead of KB, 1024 instead of 1000.
```